package.json

```diff
@@ -110,10 +119,9 @@
     "commit": "git-cz",
     "commitmsg": "commitlint --edit $HUSKY_GIT_PARAMS",
     "build-storybook": "build-storybook -c .storybook -o docs",
-    "lint": "node utils/check-lockfile-links.js && tslint --project tsconfig.json",
+    "lint": "node utils/check-lockfile-links.js && eslint -c ./eslintrc.js --ext=jsx,ts,tsx .",
     "format:prettier": "prettier --config=./.prettierrc --ignore-path=./.prettierignore \"**/*.{js,jsx,json,ts,tsx,md}\" --write",
-    "format:tslint": "tslint --project tsconfig.json --fix",
-    "format": "yarn format:prettier && yarn format:tslint",
+    "format": "yarn lint --fix",
```

Comment:
I'm a little confused. Are we removing Prettier in favor of `eslint --fix`?

Reply:
Prettier is run as part of `eslint --fix` now.

Reply:
Oh. I see. It is a eslint plugin now.